p2p/pex/addrbook.go

```diff
@@ -100,6 +101,14 @@ type addrBook struct {
 	wg sync.WaitGroup
 }
 
+var hashKey []byte
```

Comment:
is it possible to avoid creating a global var? I understand we need to seed hashKey with OS randomness, but maybe we can do it in the constructor? What do you think?